ydb/library/actors/protos/actors.proto

```diff
@@ -34,6 +34,7 @@ message TRemoteHttpInfo {
     optional bytes PostContent = 8;
     repeated THeader Headers = 9;
     optional string RemoteAddr = 7;
+    optional string UserToken = 10;
```

Comment:
we didn't have a separate UserToken field here because token is always in Headers field